src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -2155,6 +2164,22 @@ static SvgNode* _findParentById(SvgNode* node, char* id, SvgNode* doc)
 }
 
 
+static bool _checkPostponed(SvgNode* node, SvgNode* cloneNode, int depth)
+{
+    if (depth == 8192) {
```

Comment:
A recursion depth of 8192 seems excessively large.

Reply:
copied from other function in tvgSvgLoader. 
I changed it to 512